receiver/receivercreator/README.md

```diff
@@ -333,6 +334,29 @@ receivers:
           - endpoint: 'http://`endpoint`:`"prometheus.io/port" in annotations ? annotations["prometheus.io/port"] : 9090``"prometheus.io/path" in annotations ? annotations["prometheus.io/path"] : "/health"`'
             method: GET
           collection_interval: 10s
+  receiver_creator/4:
+    watch_observers: [k8s_observer]
+    receivers:
+      kubeletstats:
+        rule: type == "k8s.node"
```

Comment:
Does this add value to the specific example? Since `kubeletstats` is already defined above in `receiver_creator/3` I guess we can remove it from here?

Reply:
Yes your right. Still learning OTEL, thanks for your review :)